src/common/translation/russian-translation-set.ts

```diff
@@ -345,4 +345,8 @@ export const russianTranslationSet: TranslationSet = {
 
     controlPanel: "панель управления",
     controlPanelSettingsDescription: "Этот плагин позволяет быстро находить элементы панели управления.",
+
+    reminder: "напоминание",
+    reminderPrefix: "Префикс",
+    reminderSettingsDescription: "Этот плагин позволяет устанавливать напоминания\n",
```

Comment:
Why is `\n` neede here?

Reply:
Good catch!
It seems that when i copied and pasted from Google Translate it brought it with it.
I will fix it